_chapters/chapters/sys-lit-review.md

```diff
@@ -1,7 +1,19 @@
-# Systematic Literature Review
+---
+layout: page 
+title: Systematic Literature Review
+hide: true 
+permalink: /chapter/sys-lit-review
+---
 
 ## Introduction
 This chapter present a summary of the methodoloy to conduct a systematic literature review. First, the difference between literature review and systematic literature review is presented. This is followed by a definition of systematic literature review and a glossary of relevant terms. Finally, the phases for conducting a systematic literature review are presented as a guide. 
+Below is an interactive widget with the learning goals of this chapter. This tool allows the reader to visualize not only the learning goals but also his or her own estimation (in percent) of the achievement. 
```

Comment:
Style: I would add a new line here. So that "Below" starts with a new paragraph.